### PR TITLE
* Access purple themes

### PR DESCRIPTION
**Pull request by Wagnerp**
* Access purple themes (WIP)

image

2007:

image

2010:

image

2013 (Legacy colours):

image

365 (Legacy colours):

image

### CODE REVIEW

**Comment by Wagnerp** [comment]
Hi @Smurf-IV 

Is it ok to merge this? :)

**Comment by Smurf-IV** [comment]
> Hi @Smurf-IV
> 
> Is it ok to merge this? :)

Why are these being placed into Standard?
Why not into the themes Git repo?

**Comment by Wagnerp** [comment]
> > Hi @Smurf-IV
> > Is it ok to merge this? :)
> 
> Why are these being placed into Standard? Why not into the themes Git repo?

The new themes act like the built in themes, not XML files, but one possibility is to create a new 'Themes' package in the extended toolkit to house the new extra themes.

**Comment by Smurf-IV** [comment]
> The new themes act like the built in themes, not XML files, but one possibility is to create a new 'Themes' package in the extended toolkit to house the new extra themes.


I think that this needs to be in the themes repo tho, to try and keep that separation.

**Comment by Wagnerp** [comment]
> > The new themes act like the built in themes, not XML files, but one possibility is to create a new 'Themes' package in the extended toolkit to house the new extra themes.
> 
> I think that this needs to be in the themes repo tho, to try and keep that separation.

Will do, but there needs to be a way to 'inject' new themes based on `PaletteBase` into the `KryptonManager`. So maybe a new feature request is needed?

**Comment by Smurf-IV** [comment]
Have the purples been removed from this PR yet ?

File: Source/Krypton Components/Krypton.Toolkit/Controls Toolkit/KryptonButton.cs [hunk 1]
```diff
@@ -38,6 +38,7 @@ public class KryptonButton : VisualSimpleBase, IButtonControl, IContentValues
         private readonly PaletteTripleOverride _overridePressed;
         private IKryptonCommand _command;
         private bool _useAsDialogButton, _isDefault, _useMnemonic, _wasEnabled, _useAsUACElevationButton;
+        private float _cornerRoundingRadius;
```
Comment (Smurf-IV) [inline Source/Krypton Components/Krypton.Toolkit/Controls Toolkit/KryptonButton.cs:41]: Why is this here.. A Button already has rounding on it

File: Source/Krypton Components/Krypton.Toolkit/Controls Toolkit/KryptonCheckedListBox.cs [hunk 2]
```diff
@@ -958,6 +958,7 @@ private void LbnSelChange()
         private bool _alwaysActive;
         private bool _forcedLayout;
         private bool _trackingMouseEnter;
+        private float _cornerRoundingRadius, _itemCornerRoundingRadius;
```
Comment (Smurf-IV) [inline Source/Krypton Components/Krypton.Toolkit/Controls Toolkit/KryptonCheckedListBox.cs:961]: These rounding things should not be needed on Controls, as they already have them, AND are not part of the themes thing anyway!

File: Source/Krypton Components/Krypton.Toolkit/Controls Toolkit/KryptonButton.cs (hunk 1 above)
Comment (Wagnerp) [reply Source/Krypton Components/Krypton.Toolkit/Controls Toolkit/KryptonButton.cs:41]: Testing, part of #596, really don't need it as the value can be fed to `CornerRounding`
